.circleci/config.yml

```diff
@@ -96,11 +96,11 @@ images:
       HELM_TEST: 0
 
   - &base
-    image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    image: registry.opensuse.org/devel/openqa/ci/containers15.5/base:latest
     <<: *docker_config
 
   - &dependency_bot
-    image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    image: registry.opensuse.org/devel/openqa/ci/containers15.5/base:latest
```

Comment:
Please don't reintroduce another version string in here. As we rely on what is built in devel:openQA:ci and that repo is for this very CI use anyway we should just whatever is configured there regardless if it's 15.4, 15.5 or Tumbleweed. So just change https://build.opensuse.org/projects/devel:openQA:ci/meta when you are ready for it